### ISSUE TITLE
Supporting dynamic wsrep-node-address

### ISSUE THREAD

**Issue by wtmthethird** [mariadb-corporation/mariadb-kubernetes#44]
I was getting started using the galera configuration (with the included PR changes to support cluster IP), but I was getting issues with additional masters joining the cluster. Upon closer inspection, it was taking a guess on wsrep-node-address from the first networking interface. I found this documented here as well: http://galeracluster.com/documentation-webpages/mysqlwsrepoptions.html#wsrep-node-address - as our clusters have multiple networking interfaces per node, this does not function.

I was able to address this, but I'm not certain if it's in the best way so far, and therefore haven't submitted a PR, but I would be happy to. 

I made the following two changes to address this:
In galera.yaml, I added the following environment variable for the mariadb-server container:
```
        - name: DWAPI_PODIP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
```

Then, in the start-mariadb-kubernetes.sh script, I added an additional parameter at the end: 
`--wsrep-node-address=${DWAPI_PODIP}`

I tested this both with an undefined DWAPI_PODIP as well as it defined. While undefined, it does the normal 'guess' of the first networking interface. While defined, it properly takes in the pod's IP address, and then I am able to add subsequent masters to my cluster. However, I believe there may be a cleaner way to do this, potentially inside the init-configurations.sh container. 

Thank you!

**Comment by alexander-aldev** [comment]
Thanks for the report, I've created a Jira ticket for it.

**Comment by alexander-aldev** [comment]
Fixed in v0.1.3
